invenio_app_ils/config.py

```diff
@@ -481,7 +485,7 @@ def _(x):
         pid_type=CIRCULATION_LOAN_PID_TYPE,
         pid_minter=CIRCULATION_LOAN_MINTER,
         pid_fetcher=CIRCULATION_LOAN_FETCHER,
-        search_class="invenio_app_ils.circulation.search:IlsLoansSearch",
+        search_class=LoansSearch,
```

Comment:
This is just to unblock the PR because I don't have time to fix it right now. I have created a ticket for this: https://github.com/inveniosoftware/invenio-app-ils/issues/47.